lib/iris/tests/results/integration/name_grib/NAMEII/2_TRACER_WET_DEPOSITION.cml

```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0" ?>
```

Comment:
I think the test that produced this result file was removed a long time ago. It just so happened that I deleted all the results files in this directory when I was creating the new result files with "IRIS_TEST_CREATE_MISSING"

I have noticed there are a lot of files like this: result files where the tests that use them have been removed

But no harm in removing this now?

Reply:
@lbdreyer Yes please! It'd be great if you could purge this content now.... otherwise it'll fall between the crack, again.